BTCPayServer/wwwroot/checkout/checkout.css

```
@@ -180,6 +180,13 @@ section dl > div dd {
     margin-top: var(--btcpay-space-l);
 }
 
+.checkout-text {
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.2.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)